src/content/blog/2025-01-22-eslint-2024-year-review.md

```diff
@@ -0,0 +1,156 @@
+
+This highlights just how important sponsorships are for projects like ESLint, as a company like Tidelift can change their payout calculations and drammatically impact the health of projects.
+
+### Top sponsors
+
+In 2024, we received generous donations from many companies, led by Automattic ($24,000), Meta ($20,000), and Airbnb ($18,000). Those companies donating $1,000 or more totaled $149,027.59 USD, representing 79% of our total income.
+
+| **Sponsor**                    | **2024 Donation Total** |
+|--------------------------------|-------------------------|
+| Automattic                     | $24,000.00              |
+| Meta Open Source               | $20,000.00              |
+| Airbnb                         | $18,000.00              |
+| trunk.io                       | $7,000.00               |
+| Salesforce                     | $7,000.00               |
+| Liftoff                        | $6,000.00               |
+| Chrome Frameworks Fund         | $6,000.00               |
+| Workleap                       | $6,000.00               |
+| American Express               | $6,000.00               |
+| JetBrains                      | $5,500.00               |
+| EthicalAds                     | $5,491.08               |
+| THANKS.DEV                     | $4,234.09               |
+| Cybozu                         | $4,200.00               |
+| Canva                          | $3,000.00               |
+| Sanity                         | $2,400.00               |
+| Icons8                         | $2,400.00               |
+| Discord                        | $2,400.00               |
+| Anagram Solver                 | $2,400.00               |
+| Nx                             | $2,400.00               |
+| HeroCoders                     | $2,400.00               |
+| SERP Triumph                   | $2,000.00               |
+| Principal Financial Group      | $1,800.00               |
+| (redacted)                     | $1,741.94               |
+| Nextbase Starter Kit           | $1,612.48               |
+| Ignition                       | $1,600.00               |
+| notion                         | $1,400.00               |
+| Bitwarden                      | $1,048.00               |
+| Siemens                        | $1,000.00               |
+
+**Note:** The redacted sponsor name is from an account that was of questionable origin and was eventually blocked by GitHub. However, the money did end up in our account so we are including it in the table for transparency.
+
+### Expenses review
+
+![2024 Expenses - Summarized in following table](/assets/images/blog/2025/2024-expenses.svg)
+
+In 2024, we spent $247,669.54 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. This is in comparison to the $187,964.30 of income for a **net loss of -$59,705.24**. See the table below for an overview.
```

Comment:
```suggestion
In 2024, we spent $247,669.54 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. This is in comparison to the $187,964.30 of income for 2024 a **net loss of -$59,705.24**. See the table below for an overview.
```

The second sentence doesn't sound right, I'm guessing it's missing the year.

Reply:
Hmm, I don't think inserting the year there makes sense. The whole post is about 2024, so I don't think that's the context that's missing. What do you think is missing?

Reply:
I tried rearranging the sentences a bit. See if that is better for you.

Reply:
Sounds better now, thanks! I'll just leave another suggestion to remove a typo.